packages/manager/src/constants/API_ENDPOINTS.ts

```diff
@@ -43,7 +43,7 @@ export const API_ENDPOINTS: APIEndpoints = (() => {
 			return {
 				PrismicWroom: "https://wroom.io/",
 				PrismicAuthentication: "https://auth.wroom.io/",
-				PrismicModels: "https://customtypes.wroom.io/customtypes/",
```

Comment:
I'm not sure how this affects the `@prismicio/custom-types-client` since there were some notes there about backwards compatibility. From what I can see it shouldn't be a problem but not sure if @lihbr or @angeloashmore want to take a closer look.

See here: https://github.com/prismicio/prismic-custom-types-client/blob/main/src/client.ts#L132-L143

Reply:
Yes, it's fine!

Reply:
@danigo99 Yep, looks good to me. In case you're curious why that `if` statement was added, this PR has shares some details: https://github.com/prismicio/prismic-custom-types-client/pull/6